docs/toc.yml

```diff
@@ -32,6 +32,9 @@ items:
   - name: Explore Office JS APIs using Script Lab
     href: overview/explore-with-script-lab.md
     displayName: Excel, Word, PowerPoint
+  - name: Samples
+    href: overview/office-add-in-code-samples.md
+    displayName: PnP
```

Comment:
Should we add additional terms here to ensure that it appears as a result during TOC filtering, e.g., "code" or "code samples"?